Cubical/Foundations/Transport.agda

```diff
@@ -152,3 +152,49 @@ overPathFunct : ∀ {ℓ} {A : Type ℓ} {x y : A} (p q : x ≡ x) (P : x ≡ y)
 overPathFunct p q =
   J (λ y P → transport (λ i → P i ≡ P i) (p ∙ q) ≡ transport (λ i → P i ≡ P i) p ∙ transport (λ i → P i ≡ P i) q)
     (transportRefl (p ∙ q) ∙ cong₂ _∙_ (sym (transportRefl p)) (sym (transportRefl q)))
+
+-- substituting over a composition (P ∘ f) along p is equivalent to
+-- substituting over P along (cong f)
+-- lemma 2.3.10 in The Book
+substFunComp : ∀ {ℓ ℓ'} {A B : Type ℓ} {a a' : A}
+                 → (f : A → B) → (P : B → Type ℓ') → (p : a ≡ a') → (u : P (f a))
+                 → subst (P ∘ f) p u ≡ subst P (cong f p) u
+substFunComp f P p u = J (λ x q → (subst (P ∘ f) q u) ≡ (subst P (cong f q) u))
+  (substRefl {B = P ∘ f} u ∙ sym (substRefl {B = P} u)) p
```

Comment:
This is true by `refl` in cubical type theory, because `subst P p u` is defined to be `transport (λ i → P (p i)) u`.